packages/stark-ui/src/modules/app-sidebar/components/app-sidebar.component.html

```diff
@@ -0,0 +1,20 @@
+<mat-sidenav-container #appSidenavContainer>
+	<mat-sidenav #appSidenavLeft class="stark-app-sidenav-left" [ngClass]="{'stark-app-sidenav-menu': sidenavLeftMode==='menu'}" closed mode="over" [fixedInViewport]="true"
```

Comment:
What is the attribute "closed" used for? Is it really needed here? Shouldn't we close/open the sidebar programatically instead?